ampl-converter/src/main/java/com/powsybl/ampl/converter/version/BasicAmplExporter.java

```diff
@@ -876,6 +886,11 @@ public void writeTieLineToFormatter(TableFormatter formatter, TieLine l) throws
             int xNodeBusNum = mapper.getInt(AmplSubset.BUS, xNodeBusId);
             int xNodeVoltageLevelNum = mapper.getInt(AmplSubset.VOLTAGE_LEVEL, xnodeVoltageLevelId);
 
+            if (bus1Num == xNodeBusNum || bus2Num == xNodeBusNum) {
+                LOGGER.warn("Skipping tie line '{}' : the dangling lines of the tie line are connected to the same bus on both sides", id);
```

Comment:
Just one suggestion for the log. I think it would be good to emphasize that the problem is not on the bus but the pairing key.
```suggestion
                LOGGER.warn("Skipping tie line '{}': one of the dangling line has its bus as a pairing key.", id);
```